### PR TITLE
add continuous value model op

### PR DESCRIPTION
**Pull request by colourful-tree**
test=develop

### CODE REVIEW

File: paddle/fluid/operators/cvm_op.cc [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  }
+
+ protected:
+  // Explicitly set that the data type of computation kernel of
+  // cvm
+  // is determined by its input "X".
+  framework::OpKernelType GetExpectedKernelType(
+      const framework::ExecutionContext& ctx) const override {
+    return framework::OpKernelType(ctx.Input<Tensor>("X")->type(),
+                                   ctx.device_context());
```
Comment (chengduoZH) [inline paddle/fluid/operators/cvm_op.cc]: Now that this Op has CPU kernel, you should use `CPUPlace()` replace `ctx.device_context()` here.

Comment (colourful-tree) [reply paddle/fluid/operators/cvm_op.cc]: yeap, fix~

**Comment by shanyi15** [comment]
image
image

**Comment by shanyi15** [comment]
image
请对照上述意见修改，另外Examples的技术正确性请自查，谢谢！

**Review by chengduoZH — APPROVED** [review]

**Review by shanyi15 — APPROVED** [review]
LG comments

**Pull request merged**